cleanupJavadoc.sh

```diff
@@ -1,2 +0,0 @@
-find release/doc -name "*.html" -exec sed -i "s/.*Generated by javadoc.*//" {} \;
```

Comment:
No, it's not good, as well as keeping binary packages. Actually, there's a javadoc plugin for maven.